gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -50,6 +38,8 @@
 import java.util.Map;
 import java.util.Objects;
 
+import static com.google.gson.Gson.*;
```

Comment:
[Wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) should be not used.

Reply:
Yes, the imports were fine as they were. Probably you need to configure your IDE so it won't do this.